qutebrowser/commands/command.py

```diff
@@ -33,12 +33,15 @@ class ArgInfo:
 
     """Information about an argument."""
 
-    def __init__(self, win_id=False, count=False, flag=None, hide=False,
-                 metavar=None, completion=None, choices=None):
+    def __init__(self, win_id=False, count=False, hide=False, metavar=None,
+                 zero_count=False, flag=None, completion=None, choices=None):
         if win_id and count:
             raise TypeError("Argument marked as both count/win_id!")
+        if zero_count and not count:
+            raise TypeError("Zero_count Argument cannot exist without count!")
```

Comment:
nitpick: Please lower-case `Zero_count` (as it's a literal argument name) and `Argument` here.